ql/src/java/org/apache/hadoop/hive/ql/metadata/SessionHiveMetaStoreClient.java

```diff
@@ -2079,11 +2091,21 @@ protected GetTableResult getTableInternal(GetTableRequest req) throws TException
     Map<Object, Object> queryCache = getQueryCache();
     if (queryCache != null) {
       // Retrieve or populate cache
+      CacheKey cacheKeyTableId = new CacheKey(KeyType.TABLE_ID,req.getCatName(),req.getDbName(),req.getTblName());
+      long tableId = -1;
+
+      if(queryCache.get(cacheKeyTableId) != null)
```

Comment:
nit: need space before ( and also need to use {} for single statement block as well.